<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="resultTableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>x</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>y</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>amp</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="buttonImageScan">
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Scan</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonListClear">
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>958</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load"/>
    <addaction name="action_Save"/>
   </widget>
   <widget class="QMenu" name="menuScan">
    <property name="title">
     <string>&amp;Analyze</string>
    </property>
    <addaction name="action_Scan"/>
    <addaction name="action_Clear"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Vie&amp;w</string>
    </property>
    <widget class="QMenu" name="menuImage">
     <property name="title">
      <string>&amp;Image</string>
     </property>
     <addaction name="action_File"/>
     <addaction name="action_Last"/>
     <addaction name="action_Proc"/>
     <addaction name="action_Gray"/>
     <addaction name="action_Cmap"/>
    </widget>
    <widget class="QMenu" name="menuShow">
     <property name="title">
      <string>&amp;Show</string>
     </property>
     <addaction name="action_Cursor"/>
     <addaction name="action_Histogram"/>
     <addaction name="action_Result"/>
     <addaction name="action_Values"/>
    </widget>
    <addaction name="menuImage"/>
    <addaction name="menuShow"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>&amp;Process</string>
    </property>
    <addaction name="action_Blur"/>
    <addaction name="action_Erode"/>
    <addaction name="action_Threshold"/>
    <addaction name="separator"/>
    <addaction name="action_Reset"/>
    <addaction name="separator"/>
    <addaction name="action_Set_Threshold_Level"/>
    <addaction name="action_Set_Noise_Level"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuScan"/>
   <addaction name="menuProcess"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Load">
   <property name="text">
    <string>&amp;Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Scan">
   <property name="text">
    <string>Sc&amp;an</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Clear">
   <property name="text">
    <string>&amp;Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_File">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;File</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="action_Proc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Current</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="action_Gray">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Gray</string>
   </property>
   <property name="toolTip">
    <string>Gray</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_Cmap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Color&amp;map</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="action_Cursor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Cursor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_Histogram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Histogram</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_Result">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Result</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="action_Blur">
   <property name="text">
    <string>&amp;Blur</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_Erode">
   <property name="text">
    <string>&amp;Erode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_Threshold">
   <property name="text">
    <string>&amp;Threshold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_Reset">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Last">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Previous</string>
   </property>
   <property name="toolTip">
    <string>Last</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_Set_Threshold_Level">
   <property name="text">
    <string>Set Thres&amp;hold Level</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_Values">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Val&amp;ues</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_Set_Noise_Level">
   <property name="text">
    <string>Set &amp;Noise Level</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
